clang/test/3C/inline_anon_structs.c

```diff
@@ -5,7 +5,7 @@
 // RUN: 3c -alltypes %S/inline_anon_structs.checked.c -- | count 0
 // RUN: rm %S/inline_anon_structs.checked.c
 
-
+#include <stdlib.h>
```

Comment:
Would be better to not include `stdlib.h` if we can help it. Why is it needed?

Reply:
I added it because there's a malloc call further down, but instead I'll just grab the Checked C declaration of malloc and paste it up top.